docs-2.0/3.ngql-guide/18.operation-and-maintenance-statements/1.configs-syntax.md

```diff
@@ -0,0 +1,95 @@
+
+For example:
+
+```ngql
+nebula> SHOW CONFIGS GRAPH;
++---------+---------------------------+-------+-----------+-------+
+| module  | name                      | type  | mode      | value |
++---------+---------------------------+-------+-----------+-------+
+| "GRAPH" | "v"                       | "int" | "MUTABLE" | 0     |
++---------+---------------------------+-------+-----------+-------+
+| "GRAPH" | "minloglevel"             | "int" | "MUTABLE" | 0     |
++---------+---------------------------+-------+-----------+-------+
+| "GRAPH" | "slow_op_threshhold_ms"   | "int" | "MUTABLE" | 50    |
++---------+---------------------------+-------+-----------+-------+
+| "GRAPH" | "heartbeat_interval_secs" | "int" | "MUTABLE" | 3     |
++---------+---------------------------+-------+-----------+-------+
+| "GRAPH" | "meta_client_retry_times" | "int" | "MUTABLE" | 3     |
++---------+---------------------------+-------+-----------+-------+
+```
+
+```ngql
+nebula> SHOW CONFIGS META;
+Empty set (time spent 2059/3429 us)
+```
+
+The preceding query returns an empty set because the configurations for Meta Service are immutable in the Console.
+
+## GET CONFIGS
+
+```ngql
+GET CONFIGS [ {GRAPH|STORAGE} :] <configuration_name>
+```
+
+`GET CONFIGS` gets the specified configuration information for the specified service. When the service is not specified, Nebula Graph returns all the configurations that have the same name.
+
+For example:
+
+```ngql
+nebula> GET CONFIGS STORAGE:wal_ttl;
++-----------+-----------+-------+-----------+-------+
+| module    | name      | type  | mode      | value |
++-----------+-----------+-------+-----------+-------+
+| "STORAGE" | "wal_ttl" | "int" | "MUTABLE" | 14400 |
++-----------+-----------+-------+-----------+-------+
+```
+
+```ngql
+nebula> GET CONFIGS heartbeat_interval_secs;
++-----------+---------------------------+-------+-----------+-------+
+| module    | name                      | type  | mode      | value |
++-----------+---------------------------+-------+-----------+-------+
+| "GRAPH"   | "heartbeat_interval_secs" | "int" | "MUTABLE" | 3     |
++-----------+---------------------------+-------+-----------+-------+
+| "STORAGE" | "heartbeat_interval_secs" | "int" | "MUTABLE" | 3     |
++-----------+---------------------------+-------+-----------+-------+
+```
+
+## UPDATE CONFIGS
+
+```ngql
+UPDATE CONFIGS [ {GRAPH | STORAGE} :] <configuration_name>=<new_value>
+```
+
+`UPDATE CONFIGS` updates the specified configuration value for the specified service. `UPDATE CONFIGS` updates only the **mutable** configurations. When the service is not specified, Nebula Graph updates all the configurations that have the same name. `UPDATE CONFIGS` supports expressions.
+
+To update the **immutable** configurations, follow these steps:
+
+1. Modify the configuration files related to the configurations. The default path to the configuration file is `usr/local/nebula/etc`.
```

Comment:
/usr/local/nebula/etc

Reply:
I see in #307, `To avoid errors, we suggest that you get configurations from local.` why?
At least local_config should be false.